src/main/java/foxtrot/uniform/charlie/kilo/model/Role.java

```diff
@@ -0,0 +1,33 @@
+package foxtrot.uniform.charlie.kilo.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.springframework.security.core.GrantedAuthority;
+
+@Entity
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    @Id
+    private String name;
```

Comment:
it's better to have Long id as PK, in this case it's gonna be easier to update names